src/installer/pkg/sfx/Microsoft.NETCore.App/Microsoft.NETCore.App.Crossgen2.sfxproj

```diff
@@ -22,6 +22,8 @@
     <NativeAotSupported Condition="'$(CrossBuild)' == 'true' and '$(TargetOS)' != '$(HostOS)'">false</NativeAotSupported>
     <!-- Can't use NativeAOT in source build yet https://github.com/dotnet/runtime/issues/66859 -->
     <NativeAotSupported Condition="'$(DotNetBuildFromSource)' == 'true'">false</NativeAotSupported>
+    <ObjCopyName Condition="'$(ContinuousIntegrationBuild)' == 'true' and '$(CrossBuild)' == 'true' and '$(RuntimeIdentifier)' == 'linux-musl-arm64'">llvm-objcopy-15</ObjCopyName>
```

Comment:
@sbomer has been working on updating build containers for .NET 8 (we cannot use CentOS7 anymore since it is going to be EOL soon).

The updated containers should have recent LLVM version that will make these workarounds unnecessary once we switch to them.

Reply:
This doesn't use CentOS container. It uses Ubuntu 22.04 with Alpine Linux 3.13 rootfs.

Reply:
We are updating all build containers, not just the ones that use CentOS 7.

Reply:
This is already at the latest Ubuntu 22.04. The binutils objcopy in host doesn't support cross-build architecture, which container update will not fix. This workaround is necessary because we don't find the toolchain specific objcopy tool in NativeAOT. This is not a problem with coreclr build, because we use cmake find mechanism to locate toolchain-specific objcopy.

Reply:
Also, NativeAOT build integration is not the only place where we have this issue, it is in mono as well:

https://github.com/dotnet/runtime/blob/3e05fa1e1b4d127bd9a2c9b2e7d1d8bcb2d7a2e8/src/mono/mono.proj#L642-L667